src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -199,7 +199,7 @@ public void VolumeLabelOnNetworkOrCdRom_Throws()
                 {
                     Exception e = Assert.ThrowsAny<Exception>(() => { adrive.VolumeLabel = null; });
                     Assert.True(
-                        e is UnauthorizedAccessException || 
```

Comment:
I can remove this file. I initially made some changes that I reverted because they were not necessary, but my text editors are configured to trim those white spaces.